src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_img2img.py

```diff
@@ -68,6 +68,16 @@
 """
 
 
+# Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_img2img.produce_latents
+def produce_latents(encoder_output: BaseOutput, generator):
```

Comment:
```suggestion
def encode_to_latents(encoder_output: BaseOutput, generator):
```
(nit) would prefer to use the word "encode" here

Reply:
I suppose this may cause some confusion, because actual encoding happens outside when `vae.__call__` is executed while proposed function just extracts them from `encoder_output` (`AutoencoderKLOutput` / `AutoencoderTinyOutput`) corresponding attribute.

Reply:
You're very much correct here! I read the code too quickly. I agree, maybe just:

```suggestion
def retrieve_latents(encoder_output: BaseOutput, generator):
```

? 

Not a huge fan of the word "produce" here as it's a bit ambiguous what exactly is happening. Maybe "get" or "retrieve" is better as we're essentially just "taking" latents from the encoding

Reply:
`Retrieve` seems good to me. I'll rename funcs alongside with tests for sd1/sd2/sdxl.